sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -27,9 +27,13 @@ export interface AnalyzeSentimentResultCollection extends Array<AnalyzeSentiment
 
 // @public
 export interface AnalyzeSentimentSuccessResult extends TextAnalyticsSuccessResult {
-    documentScores: SentimentConfidenceScorePerLabel;
     sentences: SentenceSentiment[];
     sentiment: DocumentSentimentValue;
```

Comment:
I thought we were updating `DocumentSentimentValue` to `Sentiment`?
Ref -https://teams.microsoft.com/l/message/19:d6ff4003f5c848a2a2d6dcdfc0ebd497@thread.skype/1580408087458?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1580324648253&teamName=Azure%20SDK&channelName=Service%20-%20Text%20Analytics&createdTime=1580408087458

Reply:
I am not sure if we are changing the type name or just the key name. .NET is using `TextSentimentLabel` here and for the sentence sentiment, but I think that the sentiment of the document and the sentiment of the sentences are actually different types according to the API spec.